### DISCUSSION TITLE
Is `-su` and argument to `houdini:nonprofit:create`

### DISCUSSION THREAD

**Discussion by BenSturmfels** [houdiniproject/houdini#1032] [Q&A] [answered]
From the `README.md` and output of `bin/rails houdini:nonprofit:create --help`, it looks as though I should be able to specify an `-su` argument. Am I reading that right?

If I use `-su`, I get `ERROR: "rails create" was called with arguments ["-su"]`. The `--super-admin` option works though.

**Comment by clarissalimab** [comment, marked as answer]
Hello, Ben!

Thanks for reporting this problem to us. You are correct, it is not working with `-su`. Although the error is not clear on why it doesn't work, I think shortened options should have only one character so the user could combine multiple available options, that's the reason.

Would you like to send a PR to help us fixing that? We should change the alias option to `-s` instead of `-su` on [this file](https://github.com/houdiniproject/houdini/blob/main/commands/rails/commands/houdini/nonprofit/create/create_command.rb#L14).

**Reply by BenSturmfels** [reply]
Thanks Clarissa! I'd be happy to have a go at this - I haven't coded Ruby before but should be able to bumble my way through. :)

One question: that file seems to have some `desc:` definitions that are intented with 20 tabs or more. Is that meant to be? It's more visible in the raw version:

https://raw.githubusercontent.com/houdiniproject/houdini/main/commands/rails/commands/houdini/nonprofit/create/create_command.rb

**Reply by BenSturmfels** [reply]
Ah, I guess the author was just using a very small tab width. Even when tabs are set two two-space width the alignment of the `desc` key on line 15 is a bit strange... No big deal I guess.

I've now opened PR #1037.
